### PR TITLE
[SW-341] Python version independent distribution

### PR DESCRIPTION
**Pull request by mmalohlava**

### CODE REVIEW

**Comment by mdymczyk** [comment]
Nice, looks good! 👍

File: gradle/spark.gradle [hunk 1]
```diff
@@ -100,7 +100,7 @@ task printIntegTestEnv {
         println("-Dspark.testing=true\n" +
                 "-Dsparkling.assembly.jar=${project(":sparkling-water-assembly").configurations.shadow.artifacts.file.join(',')}\n" +
                 "-Dsparkling.itest.jar=${integTestJar.archivePath}\n" +
-                "-Dsparkling.pysparkling.egg=${project(":sparkling-water-py").configurations.eggs.artifacts.file.join(',')}\n" +
+                "-Dsparkling.pysparkling.egg=${project(":sparkling-water-py").configurations.sdist.artifacts.file.join(',')}\n" +
```
Comment (jakubhava) [inline gradle/spark.gradle]: `-Dsparkling.pysparkling.egg` needs to be changed to `sparkling.pysparkling.sdist` set correct variable

**Comment by jakubhava** [comment]
@mmalohlava this still need a small change - Python Integ test will fail because  `sparkling.pysparkling.egg` was replaced by `sparkling.pysparkling.sdist` however it's still not changed in here https://github.com/h2oai/sparkling-water/blob/master/py/tests/integ_test_utils.py#L40

**Comment by mmalohlava** [comment]
test

**Comment by mmalohlava** [comment]
@h2o-ops please verify!

**Comment by h2o-ops** [comment]
Hi!

**Comment by mmalohlava** [comment]
@h2o-ops please verify!

**Comment by h2o-ops** [comment]
Hi @mmalohlava, your PR title is missing JIRA issue number!

**Comment by aadu** [comment]
Any update on this?

**Comment by mli362** [comment]
No. Maybe I did something wrong? Sorry for that. I just downloaded it.

Thank you!

Best wishes,
Minming Li

**Comment by mmalohlava** [comment]
@aadu we are going to release it in the next release (in two weeks after releasing a new H2O version).

**Comment by mli362** [comment]
Thanks! I am new to H2O. I am still learning. ☺


Best wishes,
Minming Li

**Comment by aadu** [comment]
@mmalohlava I was wondering how close we are on this. I don't mean to pester you, but it's going to solve several headaches for me. Thanks

**Comment by jakubhava** [comment]
Hi @aadu, this change is going into the next sparkling water release 2.1.6. The current version is 2.1.5.

**Comment by jakubhava** [comment]
@mmalohlava maybe a good time to merge ? :)

**Comment by aadu** [comment]
Just curious if this is going to happen soon as we're on 2.1.7 now. Is there a holdup still in terms of merging this?

**Pull request merged**